e2e/support/helpers/api/createQuestionAndDashboard.ts

```diff
@@ -0,0 +1,48 @@
+import type { CardId, Dashboard, DashboardCard } from "metabase-types/api";
+
+import { createDashboard, type DashboardDetails } from "./createDashboard";
+import {
+  createQuestion,
+  type StructuredQuestionDetails,
+} from "./createQuestion";
+
+export const createQuestionAndDashboard = ({
```

Comment:
All other helpers in this PR have been moved except this one, which was extracted from a command and converted to TS.